src/components/TodoModal/TodoModal.tsx

```diff
@@ -15,30 +33,34 @@ export const TodoModal: React.FC = () => {
               className="modal-card-title has-text-weight-medium"
               data-cy="modal-header"
             >
-              Todo #2
+              {`Todo #${selectedTodo.id}`}
             </div>
 
             {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
             <button
               type="button"
               className="delete"
               data-cy="modal-close"
+              onClick={() => setSelectedTodo(null)}
             />
           </header>
 
           <div className="modal-card-body">
             <p className="block" data-cy="modal-title">
-              quis ut nam facilis et officia qui
+              {selectedTodo.title}
             </p>
 
             <p className="block" data-cy="modal-user">
               {/* <strong className="has-text-success">Done</strong> */}
               <strong className="has-text-danger">Planned</strong>
```

Comment:
delete this. 
<img width="676" alt="image" src="https://user-images.githubusercontent.com/106744410/186148643-fb2e7989-6158-4650-aba4-8bac53f5343f.png">